### PR TITLE
Modify ConnectionHandlingCommandsTest

### PR DESCRIPTION
**Pull request by sazzad16**
* Remove extending JedisCommandTestBase. This saves unnecessary setUp, tearDown.

* <strike>Remove reading from getRedisServers.</strike> Provide HostAndPort for Jedis and BinaryJedis. Only host (evidently `localhost`) is read which doesn't make complete sense. Either both host and port or none should be read.

### CODE REVIEW

**Comment by sazzad16** [comment]
@HeartSaVioR @marcosnils

File: src/test/java/redis/clients/jedis/tests/commands/ConnectionHandlingCommandsTest.java [hunk 1]
```diff
@@ -5,20 +5,19 @@
 import org.junit.Test;
 
 import redis.clients.jedis.BinaryJedis;
-import redis.clients.jedis.HostAndPort;
-import redis.clients.jedis.tests.HostAndPortUtil;
+import redis.clients.jedis.Jedis;
 
-public class ConnectionHandlingCommandsTest extends JedisCommandTestBase {
-  protected static HostAndPort hnp = HostAndPortUtil.getRedisServers().get(0);
```
Comment (HeartSaVioR) [inline src/test/java/redis/clients/jedis/tests/commands/ConnectionHandlingCommandsTest.java]: While taking only host is not valid, I think we still need to rely on this. Could we provide host and port for Jedis and BinaryJedis instance from this?

Comment (sazzad16) [reply src/test/java/redis/clients/jedis/tests/commands/ConnectionHandlingCommandsTest.java]: addressed

**Comment by sazzad16** [comment]
Merged into master, 2.10, 2.9.

**Pull request merged**